.bash_prompt

```diff
@@ -57,6 +57,13 @@ else
   sexy_bash_prompt_symbol_color="" # NORMAL
 fi
 
+# Define the default prompt terminator character '$'
+if [[ "$UID" = 0 ]]; then
```

Comment:
Per Greg's Wiki, `-eq` is true if two *integers* are equal, and is s supported by `test` / `[` so it is POSIX compatible. In bash with `[[`, both `=` and `==` are equivalent and expect a glob.

I'll make it a `==` for consistency.